js/src/interface.js

```diff
@@ -206,29 +214,51 @@ let app = new Vue({
       if (arr.length === 0 && this.showFilters && this.selectedDataSource) {
         this.addDefaultFilter();
       }
+      Vue.nextTick(() => Fliplet.Widget.autosize());
```

Comment:
@tonytlwu instead of repeating the same code on all the watchers you could simply try watching all properties like this https://github.com/vuejs/vue/issues/328#issuecomment-200917464

(I haven't tried but should work)